docs/quickstarts/ratify-on-alibabacloud.md

```diff
@@ -0,0 +1,353 @@
+# Ratify on Alibaba Cloud
+
+This guide will explain how to get up and running with Ratify on Alibaba Cloud using ACK and ACR. This will involve setting up necessary Alibaba Cloud resources, installing necessary components, and configuring them properly. Once everything is set up we will walk through a simple scenario of verifying the signature on a container image at deployment time.
+
+With Alibaba Cloud Container Service for Kubernetes (ACK) , you can quickly deploy Gatekeeper and Ratify through visual configuration. With Aliabab cloud Container Registry (ACR), you can store and distribute images with signatures together. You can use Alibaba Cloud Secrets Manager to keep your signing keys and certificates safe, and then use tools like Notation or Cosign to sign your container images with them.
+
+This article walks you through an end-to-end workflow of deploying only signed images on ACK with Ratify.
+
+
+## Table of Contents
+
+- [Prerequisites](#prerequisites)
+- [Prepare the container images in ACR](#prepare-the-container-images-in-acr)
+- [Sign ACR image](#sign-acr-image)
+    - [Sign with Notation and the keys managed in Alibaba Cloud KMS service](#sign-with-notation-and-the-keys-managed-in-alibaba-cloud-kms-service)
+        - [Configuration instructions](#configuration-instructions)
+        - [Managing KMS instances](#managing-kms-instances)
+            - [Option 1: Using KMS created and managed keys](#option-1:-using-kms-created-and-managed-keys)
```

Comment:
the link isn't rendering, try `#option-1-using-kms-created-and-managed-keys`

Reply:
Thanks Susan, fix this